indico/modules/events/client/js/reviewing/components/ChangesConfirmation.jsx

```diff
@@ -0,0 +1,68 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import {Form as FinalForm} from 'react-final-form';
+import {Form, Message, Divider} from 'semantic-ui-react';
+import {FinalTextArea} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+import './ChangesConfirmation.module.scss';
+
+export default function ChangesConfirmation() {
+  const confirmChanges = formData => {
+    console.log(formData);
+  };
+
+  return (
+    <div styleName="confirmation">
```

Comment:
I think that you don't need that at all since `Divider` already adds some nice margin.

Additional margin
![1](https://user-images.githubusercontent.com/3616940/69520024-c896fb80-0f5b-11ea-9b1f-1c47fc37dc7d.png)

Only `Divider` (without `confirmation` class)
![2](https://user-images.githubusercontent.com/3616940/69520025-c896fb80-0f5b-11ea-809d-591fd0446e49.png)